gtsfm/densify/mvsnets/methods/PatchmatchNet/datasets/data_io.py

```diff
@@ -0,0 +1,78 @@
+"""MVSDataset io utils
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
+
+"""
+
+import numpy as np
+import re
+import sys
+
+
+def read_pfm(filename):
```

Comment:
missing type hint here. maybe this is correct?
```python
def read_pfm(filename: str) -> Tuple[np.ndarray, float]:
```

Reply:
docstring too -- what does the scale represent?

Reply:
yeah.. I think I should add hints to every function first :)